Source/inv.cpp

```diff
@@ -1610,6 +1610,14 @@ void AutoGetItem(int pnum, Item *item, int ii)
 	}
 
 	if (done) {
+		if (sgOptions.Audio.bItemPickupSound && pnum == MyPlayerId) {
+			if (player.HoldItem._itype == ItemType::Gold) {
+				PlaySFX(IS_IGRAB);
+			} else {
+				PlaySFX(ItemInvSnds[ItemCAnimTbl[item->_iCurs]]);

```

Comment:
We are not supposed to play _equip_ sounds on pickup unless they are automatically equipped. It should always play `IS_IGRAB` instead.

Reply:
@julealgon oh ok, sure, I adjusted the changeset in fa71dc10ce90d6740691e37e22d05934308356a9